### ISSUE TITLE
Nicer torch summary

### ISSUE THREAD

**Issue by jeremyfix** [jeremyfix/deepcs#1]
I would like to get a summary like in keras : 
```
_________________________________________________________________
Layer (type)                 Output Shape              Param #
=================================================================
input_1 (InputLayer)         [(None, 28, 28, 1)]       0
_________________________________________________________________
flatten (Flatten)            (None, 784)               0
_________________________________________________________________
dense (Dense)                (None, 10)                7850
=================================================================
Total params: 7,850
Trainable params: 7,850
Non-trainable params: 0
_________________________________________________________________
```

rather than the one we have for the moment and taken from [this link](https://stackoverflow.com/questions/42480111/model-summary-in-pytorch) and that looks like 

```
LinearNet (
  (classifier): Linear(in_features=784, out_features=10, bias=True), weights=((10, 784), (10,)), parameters=7850
)
 7850 learnable parameters 
```

The two displays do not show the same net but I feel like the Keras like is easier to read although the torch_summarize do show other attributes (like the bias=True) not shown in the Keras display.

**Comment by jeremyfix** [comment]
see [here](https://github.com/sksq96/pytorch-summary)

**Comment by jeremyfix** [comment]
Fixed by commit 0cfb246f6e3a9f9c3276fb746c00f933d3a82650
